pkg/ebpf/Makefile

```diff
@@ -3,13 +3,13 @@ KERNEL_RELEASE ?= $(shell uname -r)
 INCLUDE_DIR = include
 
 INCLUDES =  -I $(INCLUDE_DIR)
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/arch/x86/include
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/arch/x86/include/generated
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/include
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/include/generated/uapi
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/arch/x86/include/uapi
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/include/uapi
-INCLUDES += -I /lib/modules/$(KERNEL_RELEASE)/build/arch/x86/include/asm/asm.h
+INCLUDES += -I /usr/src/kernels/$(KERNEL_RELEASE)/arch/x86/include
```

Comment:
Did you test this in a non docket context? It seems unlikely the path would change this much.

Reply:
`/lib/modules/$(KERNEL_RELEASE)/build/arch` is a symlink to `/usr/src/kernels/$(KERNEL_RELEASE)/`

Reply:
Ah yeah I remember reading that here recently. Thanks for the clarification.

Reply:
I did test it outside of docker, works fine.